css/main.css

```diff
@@ -0,0 +1,137 @@
+h1 {
+    color: #dedede;
+}
+
+p {
```

Comment:
As I said before, avoid styling directly elements, especially headings. Prefer to create a class and add it to them. It's easier to maintain a project in the long term.

Reply:
will change